h/security/permissions.py

```diff
@@ -1,4 +1,14 @@
 class Permission:
+    class Annotation:
+        ADMIN = "annotation:admin"  # Is this used anywhere?
```

Comment:
I don't think this is used anywhere. There's no concept of "administrating an annotation" that I can recall. I think we can remove this?

Reply:
Yep that's dealt with here: https://github.com/hypothesis/h/pull/6784